packages/wsjrdp2027/src/wsjrdp2027/_batch.py

```diff
@@ -742,7 +757,12 @@ def render_template(template):
     msg_date = email.utils.format_datetime(email_date)
 
     email_subject = render_template(email_subject)
+
+    if content is "":
```

Comment:
```suggestion
    if not content:
```
Das greift dann bei leerem String und bei `content is None` (im Gegensatz zu Ruby ist in Python ein leerer String false-y)